packages/grafana-ui/src/components/BarGauge/BarGauge.story.tsx

```diff
@@ -84,23 +63,54 @@ function addBarGaugeStory(overrides: Partial<Props>) {
   };
 }
 
-export const gradientVertical = addBarGaugeStory({
+export const gradientVertical: Story<Props> = addBarGaugeStory({
   displayMode: BarGaugeDisplayMode.Gradient,
   orientation: VizOrientation.Vertical,
   height: 500,
   width: 100,
 });
 
-export const gradientHorizontal = addBarGaugeStory({
+export const gradientHorizontal: Story<Props> = addBarGaugeStory({
   displayMode: BarGaugeDisplayMode.Gradient,
   orientation: VizOrientation.Horizontal,
   height: 100,
   width: 500,
 });
 
-export const lcdHorizontal = addBarGaugeStory({
+export const lcdHorizontal: Story<Props> = addBarGaugeStory({
   displayMode: BarGaugeDisplayMode.Lcd,
   orientation: VizOrientation.Vertical,
   height: 500,
   width: 100,
 });
+
+gradientVertical.args = {
+  value: 70,
+  title: 'Title',
+  minValue: 0,
+  maxValue: 100,
+  threshold1Value: 40,
+  threshold1Color: 'orange',
+  threshold2Value: 60,
+  threshold2Color: 'red',
+};
+gradientHorizontal.args = {
+  value: 70,
+  title: 'Title',
+  minValue: 0,
+  maxValue: 100,
+  threshold1Value: 40,
+  threshold1Color: 'orange',
+  threshold2Value: 60,
+  threshold2Color: 'red',
+};
+lcdHorizontal.args = {
+  value: 70,
+  title: 'Title',
+  minValue: 0,
+  maxValue: 100,
+  threshold1Value: 40,
+  threshold1Color: 'orange',
+  threshold2Value: 60,
+  threshold2Color: 'red',
+};
```

Comment:
These seems to be all set to the same value, you can just do 
gradientVertical.args = gradientHorizontal.args = lcdHorizontal.args